cli/command/node/list_test.go

```diff
@@ -160,3 +162,23 @@ func TestNodeListFormat(t *testing.T) {
 	assert.Contains(t, buf.String(), `nodeHostname1: Leader`)
 	assert.Contains(t, buf.String(), `nodeHostname2: Reachable`)
 }
+
+func TestNodeListOrder(t *testing.T) {
+	buf := new(bytes.Buffer)
+	cli := test.NewFakeCli(&fakeClient{
```

Comment:
This function has changed on master. It no longer takes a second argument, but defaults to using a buffer.

You can access the buffer from `cli.OutBuffer()`